docs/spec/UserGroups.schema.json

```diff
@@ -13,9 +13,10 @@
         "items": {
           "$ref": "User.schema.json"
         }
-      }
+      },
+      "can_remove": {"type": "boolean"}
     },
     "additionalProperties": false,
-    "required": ["name", "members"]
+    "required": ["name", "members", "can_remove"]
```

Comment:
This is actually the schema for the `UserGroupViewModel`. This isn't part of the public API spec, rather just here to facilitate testing of the internal API method. Is there a need to distinguish models returned from internal API calls from models that are used to render templates? If so neither `UserGroupViewModel` nor `ReportReaderViewModel` are view models in that sense...

Reply:
But actually it's true that this `can_remove` property isn't needed here, because it'll be obvious from context (i.e. which vue component retrieves it) whether or not it should be removable, so will remove altogether.

Reply:
I guess technically these are "data transfer objects" rather than "view models". Shall we distinguish DTOs from VMs by naming convention?